addons/event_sale/models/event.py

```diff
@@ -6,6 +6,15 @@
 from openerp.osv import fields as old_fields
 
 
+class event_type(models.Model):
```

Comment:
It should be inherited from product.template, so when you create a new event type it will create a new product with same name and event_ok fields set to True by default

remove code related to event_type_id from everywhere.